code/modules/mob/living/carbon/human/human_defines.dm

```diff
@@ -53,3 +53,6 @@
 	var/name_override //For temporary visible name changes
 
 	var/heart_attack = 0
+
+	var/drunkenness = 0 //Overall drunkenness - check handle_alcohol() in life.dm for effects
+	var/drunk_state = NOT_DRUNK //A mob's state of drunkenness

```

Comment:
You don't need two variables for this. Just use drunkness and make defines for threshold values.